spec/views/events/participants.html.erb_spec.rb

```diff
@@ -113,4 +113,16 @@
     expect(rendered).to have_css('button#open_print_modal')
     expect(rendered).to have_css('input#print_participant_list')
   end
+
+  it "displays an modal that allows selection of email target" do
+    render
+    expect(rendered).to have_css('div#send_participant_email_modal')
+  end
+
+  it "allows selection of participants in the modal" do
+    application_letter = FactoryGirl.create(:application_letter, user: FactoryGirl.create(:user_with_profile, role: :admin), event: @event, status: 1)
```

Comment:
Why does the user have to be an admin?
You can use the `accepted_application_letter` factory instead of setting the status.